src/Profiles/MuscleOxygen/DataPages/Base/ANTPLUS_MuscleOxygenBaseMainDataPageMsg.h

```diff
@@ -0,0 +1,24 @@
+#ifndef ANTPLUS_MUSCLEOXYGENBASEMAINDATAPAGEMSG_h
+#define ANTPLUS_MUSCLEOXYGENBASEMAINDATAPAGEMSG_h
+
+#include <BaseClasses/ANTPLUS_BaseDataPageMsg.h>
+#include <Profiles/MuscleOxygen/ANTPLUS_MuscleOxygenPrivateDefines.h>
+
+#include "ANT.h"
+
+class MuscleOxygenBaseMainDataPageMsg : public BaseDataPageMsg<BroadcastDataMsg> {
+public:
+    MuscleOxygenBaseMainDataPageMsg(uint8_t dataPageNumber = ANTPLUS_MUSCLEOXYGEN_DATAPAGE_MUSCLEOXYGENDATA_NUMBER);
```

Comment:
I think we can skip the base class here as well since we only have 1 page that isn't globally common :)